spec/miscellaneous/helpers/mailer_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
```

Comment:
Should we turn on Hound's double quote nagging?

Reply:
Nah, it gets super annoying -- that's why it's not currently on. Now that I know these aren't always single quotes a A Reason, I won't make this mistake again.